cypress/support/fragments/check-in-actions/checkInActions.js

```diff
@@ -49,6 +50,10 @@ const actionsButtons = {
   printHoldSlip: printHoldSlipButton,
 };
 const itemAnumberOfPieces = '2';
+const dateField = TextField({ id: 'dp-4' });
```

Comment:
It seams that it's autogenerated id and it can be different on various environments